tasks/all-contributors/.yarnrc.yml

```diff
@@ -0,0 +1,8 @@
+yarnPath: "../../.yarn/releases/yarn-3.0.0.cjs"
```

Comment:
Settings are inherited from the parent folder so you probably don't need this config file

Reply:
Do you have any documentation about how this works? The `tasks/` directory is not a workspace in this repo setup. So wasn't sure how/if each of these should be handled.

Very helpful.

Reply:
It's mentioned in the docs at the top here https://yarnpkg.com/configuration/yarnrc
> [...] Yarn will automatically find them in the parent directories [...]